.vscode/settings.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Please keep this directory private. I/We don't need this. I have my very own settings too but I don't publish them.

Reply:
I can add it to `.gitignore` if it's ok with you...

Reply:
Yes, please! The whole `.vscode` directory, please 😄 !

Reply:
I'll remove the folder later...

I can't remember from the top of my head how to untrack it/purge it from commit.

But it is added to .gitignore now

Reply:
```bash
git rm -r --cached .vscode/
git commit -a -m "Some message"
git push
```

Was the simple answer :yum: